com.microsoft.mrtk.input/Interactors/Poke/PokeInteractor.cs

```diff
@@ -201,12 +177,11 @@ public override void ProcessInteractor(XRInteractionUpdateOrder.UpdatePhase upda
                     // The start of our new trajectory is the end of the last frame's trajectory.
                     pokeTrajectory.Start = pokeTrajectory.End;
 
-                    // If we can get a joint pose, set our attachTransform accordingly.
-                    // pokePointTracked sets isHoverActive.
```

Comment:
nit: this comment still seems useful, to give context on why we're caching this value

Reply:
Added back in!